sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/Sonar_way_profile.json

```diff
@@ -112,6 +112,7 @@
     "S6532",
     "S6558",
     "S6611",
-    "S6615"
+    "S6615",
+    "S6622"
```

Comment:
Remove this rule from the SonarWay, as this rule implementation shouldn't be a part of this PR

Reply:
Fixed.